### PR TITLE
Sql server add to project guide

### PR DESCRIPTION
**Pull request by molebox**
This PR adds a new "Add to existing project" guide for SQL Server

### CODE REVIEW

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx [hunk 1]
```diff
@@ -105,6 +105,29 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
...
+<SwitchTech technologies={['*', 'sqlserver']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "sqlserver"
+  url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](../../../../concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) which is defined in `.env`:
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx]: Has double colon at end of senteence, but then another sentence before actualy example

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx [hunk 2]
```diff
@@ -111,6 +111,29 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
...
+ url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](../../../../concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) which is defined in `.env`:
+
+The following example connection URL [uses SQL authentication](../../../../concepts/database-connectors/sql-server/sql-server-local#enable-authentication-with-sql-logins-optional), but there are [other ways to format your connection URL](../../../../concepts/database-connectors/sql-server/sql-server-connection-string)
+
+```bash file=prisma/.env
+  DATABASE_URL="sqlserver://localhost:1433;database=mydb;user=sa;password=r@ndomP@$$w0rd;trustServerCertificate=true"
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx]:
```suggestion
DATABASE_URL="sqlserver://localhost:1433;database=mydb;user=sa;password=r@ndomP@$$w0rd;trustServerCertificate=true"
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx [hunk 3]
```diff
@@ -317,7 +317,7 @@ If you need the relation to hold other fields, you can create an [explicit](many
 <TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>]}>
 <tab>
 
-```prisma highlight=5,6;normal
+```prisma
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx:320]: This looks unrelated, buuuut: Is this correct? Why remove the highlight completely?

**Review by janpio — APPROVED** [review]
Left a few comments, besides that great!

File: content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx (hunk 3 above)
Comment (molebox) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx:320]: This is from another PR
